pkg/action/install.go

```diff
@@ -737,20 +737,22 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 	name = strings.TrimSpace(name)
 	version := strings.TrimSpace(c.Version)
 
-	if _, err := os.Stat(name); err == nil {
-		abs, err := filepath.Abs(name)
-		if err != nil {
-			return abs, err
-		}
-		if c.Verify {
-			if _, err := downloader.VerifyChart(abs, c.Keyring); err != nil {
-				return "", err
+	if c.RepoURL == "" {
+		if _, err := os.Stat(name); err == nil {
+			abs, err := filepath.Abs(name)
+			if err != nil {
+				return abs, err
 			}
+			if c.Verify {
+				if _, err := downloader.VerifyChart(abs, c.Keyring); err != nil {
+					return "", err
+				}
+			}
+			return abs, nil
+		}
+		if filepath.IsAbs(name) || strings.HasPrefix(name, ".") {
```

Comment:
Should we still check for an absolute or relative path and error out when `--repo` is specified?

Reply:
It doesn't make sense to me to check the path when RepoURL is provided as `--repo` only accepts something remote with a protocol (http, https, s3, ...), the only thing I can see will be different is that with this change, it may allow a chart name in a remote repository to start with `.`, but not sure how much an issue that could be.

Reply:
You're right. I was just thinking about a user error in specifying the arguments, but then the error message we currently have wouldn't even be correct.  Let's just leave it as you have it.